packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 53.14.0 - January 19, 2022
+
+## Fixed
+
+#### salesforcedx-utils-vscode
+
+- Added replay debugging to ApexLibraryExecuteExecutor ([PR #3758](https://github.com/forcedotcom/salesforcedx-vscode/pull/3758))
+
+#### salesforcedx-visualforce-language-server
+
+- Chore(deps-dev): bump shx from 0.2.2 to 0.3.4 ([PR #3754](https://github.com/forcedotcom/salesforcedx-vscode/pull/3754))
```

Comment:
@RitamAgrawal let's remove the dependency updates from the change log, so we don't need this or the next line.